cypress/e2e/visit-homepage.cy.js

```diff
@@ -0,0 +1,21 @@
+describe('Visit homepage', () => {
+  it('visit homepage QA', {
```

Comment:
This implementation is fine but could be made cleaner and more maintainable by storing this config in a constant and passing it as an argument.